website/content/api-docs/secret/identity/group.mdx

```diff
@@ -4,6 +4,11 @@ page_title: 'Identity Secret Backend: Group - HTTP API'
 description: This is the API documentation for managing groups in the identity store.
 ---
 
+~> **NOTE:** Be careful in granting permissions to non-readonly identity endpoints.
+If a user can modify an entity, they can grant it additional privileges through
+policies.  If they can modify an alias they can login with, they can bind it to
+an entity with higher privileges.
```

Comment:
Maybe?
```suggestion
an entity with higher privileges as long as they have access to the entity with more privileges.
```

Reply:
Not sure what you're getting at here?

Reply:
The first time I read it my initial reading included some "they can bind it to ANY entity with higher privileges" and not "they can bind it to an entity they already have access to". This suggestion definitely gets into areas where my understanding is kind of hazy though, so I don't mind if you deem it nonsensical.

Reply:
It is the former, i.e. ANY entity.  That's the motivation for this warning.